waffle-compiler/test/flattener/flattenFiles/parent.sol

```diff
@@ -0,0 +1,75 @@
+// Dependency file: test/flattener/testSource/cycle1.sol
```

Comment:
Shouldn't `waffle-compiler/test/flattener/flattenFiles/` be in the .gitignore? 
It seems to be temporary test output.